types/url-assembler/index.d.ts

```diff
@@ -1,19 +1,22 @@
-// Type definitions for url-assembler 1.2
+// Type definitions for url-assembler 2.1.0
```

Comment:
```suggestion
// Type definitions for url-assembler 2.1
```

To fix the CI failure; took me a bit to work out where to find the actual log but it looks like the error is:

> ERROR: 1:25  dt-header  Error parsing header. Expected: foo MAJOR.MINOR (patch version not allowed). See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md

Reply:
ahh thanks for finding the issue! just pushed the change